app/models/project.rb

```diff
@@ -31,6 +31,7 @@ class Project < ActiveRecord::Base
   belongs_to :user
   belongs_to :category
   has_and_belongs_to_many :channels
+  has_and_belongs_to_many :plans, join_table: 'projects_plans'
```

Comment:
You have no tests in your model spec for these changes about has_and_belogns_to_many relation.
```ruby
it { is_expected.to have_and_belong_to_many(:plans)
```